### ISSUE TITLE
Actionlib clients connecting to actionlib server via rosbridge_server websocket

### ISSUE THREAD

**Issue by mcross8** [RobotWebTools/rosbridge_suite#124]
Ubuntu 12.04 / ROS Hydro

**TL;DR**: 2 different actionlib_clients connected to actionlib_servers via rosbridge_server websocket result in differing behaviour.

Hello, here's my setup: 1 Kinova MICO robotic manipulator arm; 2 Actionlib servers (1 for angular control and 1 for position control); 2 different Actionlib clients (both clients send action goals to the action server via a rosbridge_server websocket).

The first action_client is a simple html file with the roslibjs library calls, based on examples on the ROS wiki. The second action_client sends stings containing JSON-formatted data to the rosbridge_server websocket.

I'm currently using the JACO packages, which have the Actionlib servers baked into jaco_arm_driver node. The node launches both action servers. The two action servers are for angular control and position control. Let's say I have goals for each Actionlib Server: Angle1, Angle2, Pose1, Pose2. Let's also say that each goal results in different arm configurations. 

The issue I am encountering has to do with the actionlib server behavior in reaction to goals sent via the rosbridge_server websocket. The above noted clients which both send JSON-formatted data strings each encounter different problems.

For example, client 1 (which calls roslibjs) can send the following sequence of goals:
Angle1 -> Angle2 -> Pose1 -> Pose2
All actions are executed correctly. However, when client 1 sends the following sequence:
Angle1 -> Pose1 -> Angle1 
The second instance of Angle1 is not processed.

Client 2 (which doesn't call roslibjs) can send the following sequence of goals:
Angle1 -> Pose1 -> Angle1 
And those actions are executed correctly. However, when client 2 sends the following srequence:
Angle1 -> Angle2 -> Pose1 -> Pose2
Neither Pose1 nor Pose2 will process.

Both sets of behaviour are repeatable with their respective clients.

At first I belived this to be an actionlib issue; however, since the issue is different depending on the client used, my next best guess is the rosbridge_server and/or websocket and/or roslibjs.

Have you experienced issues related to sending actionlib goals via rosbridge? Is there something really, really obvious I'm missing? 

Thanks very much!
